src/cascadia/TerminalSettingsModel/CascadiaSettings.h

```diff
@@ -156,6 +156,9 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
         void ExpandCommands();
 
+        static winrt::event_token AzureOpenAISettingChanged(const AzureOpenAISettingChangedHandler& handler);

```

Comment:
Can this be a `static til::event`? I dunno if I've ever done a _static_ event so I genuinely don't know

Reply:
Check out `ConptyConnection::NewConnection`!